test/js_interop_helpers_test/shared_tests.dart

```diff
@@ -19,28 +21,34 @@ void verifyJsFileLoaded(String filename) {
 
 void sharedJsFunctionTests() {
   group('JS functions:', () {
+    // TODO: remove in 5.0.0
```

Comment:
Are we able to deprecate `getProperty` in the current master line _(is there an alternative for consumers)_? Or do we need to deprecate in 5.0.0, and remove in 6.0.0?

Reply:
It appears that `dart:js_util` has `getProperty`,`setProperty`, `newObject`, `hasProperty` and a few other fancy helpers we can use instead. Maybe i should just map all ours to those instead and then add a deprecation message to use that instead?

Reply:
see https://github.com/cleandart/react-dart/pull/160#discussion_r262942145